imagestreams/golang-rhel7.json

```diff
@@ -0,0 +1,52 @@
+{
+    "apiVersion": "v1",
+    "kind": "ImageStream",
+    "metadata": {
+        "annotations": {
+            "openshift.io/display-name": "Go"
+        },
+        "name": "golang"
+    },
+    "spec": {
+        "tags": [
+            {
+                "annotations": {
+                    "description": "Build and run Go applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/golang-container/blob/master/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Go available on OpenShift, including major version updates.",
+                    "iconClass": "icon-go-gopher",
+                    "openshift.io/display-name": "Go (Latest)",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "sampleRepo": "https://github.com/sclorg/golang-ex.git",
+                    "supports": "golang",
+                    "tags": "builder,golang,go"
+                },
+                "from": {
+                    "kind": "ImageStreamTag",
+                    "name": "1.11.5"
+                },
+                "name": "latest",
+                "referencePolicy": {
+                    "type": "Local"
+                }
+            },
+            {
+                "annotations": {
+                  "description": "Build and run Go applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/golang-container/blob/master/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Go available on OpenShift, including major version updates.",
+                  "iconClass": "icon-go-gopher",
+                  "openshift.io/display-name": "Go (1.11.5)",
+                  "openshift.io/provider-display-name": "Red Hat, Inc.",
+                  "sampleRepo": "https://github.com/sclorg/golang-ex.git",
+                  "supports": "golang",
+                  "tags": "builder,golang,go"
+                },
+                "from": {
+                    "kind": "DockerImage",
+                    "name": "registry.redhat.io/devtools/go-toolset-rhel7:1.11.5"
```

Comment:
Why a specific tag is used here? that would mean the imagestream needs to be updated for every new build? I'd simply use :latest or no tag at all (if valid).

Reply:
@jcajka or is the 1.11.5 something that would not change?

Reply:
Or put another way -- what happens if golang 1.11.6 is available -- should that be automatically updated for users or not? Maybe it's correct in the end -- if so, sorry for confusion...

Reply:
hm... that is good point, IMHO here it would make sense to have tag for 1.11(minor updates shouldn't matter) or something a like, but unfortunately I don't know what tagging schema is RHEL golang image using and this actually might be correct.

Reply:
This is a problem with the golang image tagging. I can map it here to hide if it makes sense, such as :1.11 -> 1.11.5 or 1.11.6 but I was just trying to get this image-stream available. There is another thread about improving the tagging.

I think it makes sense to have these floating tags:
`latest`
`1.11`
but not `1` but I'm not expert on the promise of compatibility between the minor numbers.

These issues seem beyond this PR, which is to just wrap what the image does for now.

Reply:
@sspeiche I do agree we can always change it.